sway/server.c

```diff
@@ -49,6 +49,9 @@
 #include "sway/xwayland.h"
 #endif
 
+#define SWAY_XDG_SHELL_VERSION 2
+#define SWAY_LAYER_SHELL_VERSION 4
```

Comment:
To fix https://github.com/swaywm/sway/issues/7260, we need to advertise 3 here.

Reply:
Done